Formula/a/agda.rb

```diff
@@ -86,37 +96,30 @@ class Agda < Formula
 
   def install
     agda2hs = buildpath/"agda2hs"
-    agdalib = lib/"agda"
+    agdalib = pkgshare # for write permissions needed to re-generate .agdai when using different options
     cubicallib = agdalib/"cubical"
     categorieslib = agdalib/"categories"
 
+    # Add a backwards compatibility symlink. Can consider removing in a future release
+    lib.install_symlink pkgshare
```

Comment:
```suggestion
    lib.install_symlink pkgshare => "agda"
```

Otherwise you end up not actually backwards compatible in the name of the link!

Reply:
By default, the basename (i.e. `File.basename(pkgshare)`) is used, which should already be `agda`:
```rb
❯ brew ruby -e 'p File.basename(Formula["agda"].pkgshare)'
"agda"
```

Reply:
Didn’t know that!  Carry on then; thank you!